<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>swc_matrixInput</class>
 <widget class="QDialog" name="swc_matrixInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>143</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="ker00">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker01">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker02">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker21">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ker22">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter 3x3 Kernel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
